{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-18704.json"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "last_affected": "077c0ae3fc724dca10b648add7d2758451549996"
            }
          ],
          "repo": "https://github.com/fusionbox/django-widgy",
          "type": "GIT"
        }
      ],
      "versions": [
        "0.4.0",
        "0.5.0",
        "0.6.0",
        "0.6.1",
        "0.7.0",
        "0.7.1",
        "0.8.3",
        "0.8.4",
        "v0.1.0",
        "v0.7.2",
        "v0.7.4",
        "v0.8.1",
        "v0.8.2"
      ]
    }
  ],
  "aliases": [
    "GHSA-98hv-qff3-8793",
    "PYSEC-2021-336"
  ],
  "details": "Unrestricted Upload of File with Dangerous Type in Django-Widgy v0.8.4 allows remote attackers to execute arbitrary code via the 'image' widget in the component 'Change Widgy Page'.",
  "id": "CVE-2020-18704",
  "modified": "2025-01-14T08:36:04.346589Z",
  "published": "2021-08-16T18:15:07Z",
  "references": [
    {
      "type": "REPORT",
      "url": "https://github.com/fusionbox/django-widgy/issues/387"
    }
  ],
  "schema_version": "1.6.0",
  "severity": [
    {
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
      "type": "CVSS_V3"
    }
  ]
}